[EDU-2771] docs: improve old functions section in Rules Engine (#418)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx

<Button href="/en/documentation/products/edge-application/domains/mtls/" text="Learn more about support for mTLS"></Button>

## Variables as arguments

> For behaviors that require an argument, you can use the same variables available for the processing phase in which they are available. This way, you can, for example, compose cookies or HTTP headers using data collected durin the request phase, such as the user's device group or their geolocation.

Reviewer: ```suggestion
For behaviors that require an argument, you can use the same variables available for the processing phase in which they're available. This way, you can, for example, compose cookies or HTTP headers using data collected during the request phase, such as the user's device group or their geolocation.
```

For example, for the following rule in the **Response Phase**:

> If `${host}` *is equal* `myhost.com`
> Then **Add Response Cookie** `cookie-host-value=${host}`

When this rule is active, if the request is made from the host determined in the criteria, the response will return the following header: `Set-Cookie: cookie-host-value=myhost.com`.

Azion also provides special variables that act as functions and take arguments. The following variables can be used to compose **Behavior** arguments:

| Variable | Description |
| --- | --- |

> | `${cookie_time_offset(number)}` | Returns the current date plus an offset in seconds, entered as an argument, to be used to define the expiration time of a cookie. For example, if you want the cookie to expire in 1 hour from the time it is created, use the *Add Response Cookie* behavior with the argument: `cookie-name=cookie-value; Expires=${cookie_time_offset(3600)}` |

Reviewer: ```suggestion
| `${cookie_time_offset(number)}` | Returns the current date plus an offset in seconds, entered as an argument, to be used to define the expiration time of a cookie. For example, if you want the cookie to expire in 1 hour from the time it's created, use the *Add Response Cookie* behavior with the argument: `cookie-name=cookie-value; Expires=${cookie_time_offset(3600)}` |
```
